bin/hoc-2015-survey/export-survey-results

```diff
@@ -26,10 +27,57 @@ columns = %w[
   event_improvement_s
 ]
 
+# de-dupe repeat responses, take the latest valid one
+# skip forms with 10k+ or non-integer students (any of the 3 columns)
+# skip forms with more girls or minorities than total students
+MAX_STUDENTS = 10000
+
+emails = Set.new
+dupes = 0
+skipped = 0
+skipped_emails = Set.new
+
+# Empty string and nil return 0. Otherwise it should be an int or a string that parses to an int.
+def parse_int_or_nil(value)
+  return 0 if value.nil? || (value.is_a?(String) && value.empty?)
```

Comment:
you could do just value == '', ruby doesn't do any wat stuff with the == operator